src/PrestaShopBundle/Resources/views/Admin/Sell/CustomerService/CustomerThread/Block/list_header_html.twig

```diff
@@ -0,0 +1,74 @@
+{# **
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * #}
+
+{# {% extends file="helpers/list/list_header.tpl" %} #}
+
+{% block leadin %}
+  <div class="">
+    <div id="CustomerThreadContacts" class="d-flex justify-content-around">
+      {% set nb_categories = categories|length %}
+      {% for key, val in categories %}
+
+        {% set id_customer_thread = 0 %}
+        <div class="col-lg-3 p-0 card">
+          <div class="card-header">
+            {{ val.name }}
+          </div>
+          <div class="card-body">
+
+            {% if nb_categories < 6 %}
```

Comment:
Why do you display the description only if there are less than 6 categories? 🤔 This is already inside a card so I don't think it will impact the display anyway
Besides from the screenshot you posted it seems like you only have two service summaries right? So you should drop this condition

Reply:
You can add more of them, and the logic to show if its less then 6 was before i kept it but yeah it doesn't make much sense i guess so i removed it. And improved the display
![image](https://user-images.githubusercontent.com/51944825/205245410-2d8d2546-a975-4ae6-b848-50c9b52183f5.png)